tests/fixtures/logs.py

```diff
@@ -24,3 +28,82 @@ def gelf_logger():
     logger.addHandler(handler)
 
     return logger
+
+
+def server_event(event_number):
+    """Returns event_number of events of event_type server"""
+    return pd.DataFrame(
+        data=[
+            {
+                "username": FAKE.profile().get("username"),
+                "event_type": FAKE.uri_path(),
+                "ip": FAKE.ipv4_public(),
+                "agent": FAKE.user_agent(),
+                "host": FAKE.hostname(),  # not really as the host is the docker container ID
+                "referer": FAKE.url(),
+                "accept_language": FAKE.locale(),  # could be also something like "en-US,en;q=0.5"
+                "event": '{"POST": {}, "GET": {}}',
+                "event_source": "server",
+                "context": {
+                    "course_user_tags": {},
+                    "user_id": FAKE.random_digit_or_empty(),
+                    "org_id": FAKE.word(),
+                    "course_id": "{}+{}+{}".format(
+                        FAKE.word(), FAKE.word(), FAKE.word()
+                    ),
+                    "path": FAKE.uri_path(),  # "event_type" and "context:path" share the same URI
+                },
+                "time": FAKE.iso8601(),
+                "page": "null",
+            }
+            for i in range(event_number)
+        ]
+    )
+
+
+def browser_event(event_number):
+    """Returns event_number of events of event_type browser"""
+    return pd.DataFrame(
+        data=[
+            {
+                "username": FAKE.profile().get("username"),
+                "event_source": "browser",
+                "name": "page_close",
+                "accept_language": FAKE.locale(),  # could be also something like "en-US,en;q=0.5"
+                "time": FAKE.iso8601(),
+                "agent": FAKE.user_agent(),
+                "page": FAKE.url(),
+                "host": FAKE.hostname(),  # not really as the host is the docker container ID
+                "session": FAKE.md5(raw_output=False),
+                "referer": FAKE.url(),
+                "context": {
+                    "user_id": FAKE.random_digit_or_empty(),
+                    "org_id": FAKE.word(),
+                    "course_id": "{}+{}+{}".format(
+                        FAKE.word(), FAKE.word(), FAKE.word()
+                    ),
+                    "path": "/event",
+                },
+                "ip": FAKE.ipv4_public(),
+                "event": "{}",
+                "event_type": "page_close",  # could be an other sting too
+            }
+            for i in range(event_number)
+        ]
+    )
```

Comment:
That's a good first approach, but I think we should go deeper and handle different edx's server or browser event types, _e.g._ problem check, etc.

Reply:
Thanks for the review Julien! Yes, there are more events to add, working on it